File: sorting_algorithm_python/bubblesort.py
arr=[2,3,4,6,7,1,8,9,10,100,400,50,70,]

def bubblesort(arr):
    for i in range(0,len(arr)-1):
        swap=False
        for j in range(0,len(arr)-1):
            if(arr[j]>arr[j+1]):
                arr[j],arr[j+1]=arr[j+1],arr[j]
                swap=True
        if(swap==False):
            break

    print(arr)

bubblesort(arr)

File: sorting_algorithm_c/bubblesort.c
#include<stdio.h>
#define size 10
#include<stdbool.h>

void swap(int *l,int *f){
    int temp;
    temp=*l;
    *l=*f;
    *f=temp;
}

void bubblesort(int arr[size],int n){
    int i,j;
    

    for(i=0;i<n-1;i++){
        int swaps=0;
        for(j=0;j<n-1-i;j++){
            if(arr[j+1]<arr[j]){
                swap(&arr[j+1],&arr[j]);
                swaps=1;

            }
            
        }
        if(swaps==0){
            break;
        }
    }

}

void print(int arr[size],int n){
    int i;
    for(i=0;i<n-1;i++){
        printf("%d ",arr[i]);

    }
}



int main(){
    int arr[size];
    int i;

    printf("Enter the array elements");
    for(i=0;i<size-1;i++){
        scanf("%d",&arr[i]);
    }
    bubblesort(arr,size);
    print(arr,size);




}
